src/qt/bitcoinamountfield.cpp

```diff
@@ -179,6 +164,25 @@ class AmountSpinBox: public QAbstractSpinBox
         return QAbstractSpinBox::event(event);
     }
 
+    StepEnabled stepEnabled() const
+    {
+        StepEnabled rv = 0;
+        if (isReadOnly()) // Disable steps when AmountSpinBox is read-only
+            return StepNone;
+        if(text().isEmpty()) // Allow step-up with empty field
```

Comment:
Haha :), got me. Well the only reason I did not add a space was because the surrounding if clauses also have the wrong style ^^.